addons/project/static/src/views/project_form/project_form_renderer.js

```diff
@@ -1,9 +1,17 @@
 /** @odoo-module */
 
 import { FormRendererWithHtmlExpander } from "../form_with_html_expander/form_renderer_with_html_expander";
+import { Chatter } from "@mail/web/chatter";
+import { ProjectSettings } from "./project_settings";
 
 export class ProjectFormRenderer extends FormRendererWithHtmlExpander {
     get htmlFieldQuerySelector() {
         return '.o_field_html[name="description"]';
     }
 }
+
+ProjectFormRenderer.components = {
+    ...ProjectFormRenderer.components,
+    Chatter,
```

Comment:
Why do we have to add the `chatter` component?

Reply:
> Why do we have to add the `chatter` component?

A chatter error raised  so I added. today  I checked it resolved, so there is no need. thank you